configs/mvp/campus_config/mvp_campus.py

```diff
@@ -171,16 +171,51 @@
         ),
     ),
     evaluation_setup=dict(
-        type='MVPEvaluation',
+        type='End2EndEvaluation',
         dataset_name=__test_dataset__,
         pred_kps3d_convention='campus',
         gt_kps3d_convention='campus',
+        eval_kps3d_convention='campus',
         n_max_person=__n_instance__,
-        selected_limbs_name=[
-            'left_lower_leg', 'right_lower_leg', 'left_upperarm',
-            'right_upperarm', 'left_forearm', 'right_forearm', 'left_thigh',
-            'right_thigh'
+        checkpoint_select='pcp_total_mean',
+        metric_list=[
+            dict(
+                type='PredictionMatcher',
```

Comment:
`PredictionMatcher` is not a metric but is the first step for almost every metric. Since no other classes need the results from `PredictionMatcher`, it is more accessible to set the matcher as rank-0 in the metric list to reuse the matching results for other metrics. Removed "metric" from the docstrings for `PredictionMatcher` in [2342328](https://github.com/openxrlab/xrmocap/pull/79/commits/2342328e10b30abba07cf90b328d1a78375812cb).